freqtrade/optimize/backtesting.py

```diff
@@ -25,6 +25,7 @@
 from freqtrade.exceptions import DependencyException, OperationalException
 from freqtrade.exchange import (amount_to_contract_precision, price_to_precision,
                                 timeframe_to_minutes, timeframe_to_seconds)
+from freqtrade.freqai import freqai_util
```

Comment:
I'm a bit concerned about this import.

the assumption is that freqAI is an optional module (just as hyperopt) - and all it's dependencies are optional as well.
While i think it works now - importing this here creates a direct dependency on the `freqtrade.freqai` package to always resolve / load (even with freqAI dependencies missing).

one small change (which will not be seen in the context of this import), adding an import to `freqai/__init__.py` can break this - which is something that'll not be captured by tests (we require all dependencies for tests).

If we need this import, it should be moved to a "freqAI vettet" area - which we know is only running when freqAI is installed / used.
maybe look at the imports in `interface.py` for ideas (basic idea is to move the import to the function that uses the stuff - which is behind the "paywall" of "freqai.enabled=true").

Reply:
This is resolved.